### DISCUSSION TITLE
help with usage

### DISCUSSION THREAD

**Discussion by DevSamuelOliveira** [kevinzg/facebook-scraper#464] [Q&A] [answered]
First I apologize if the communication doesn't flow, I'm using google translator.

I'm a student in the area and I need to cut some facebook data for my job, would I have a basic guide on how to use the system?

my first difficulty is:

C:\Python39\lib\site-packages\facebook_scraper\facebook_scraper.py:437: UserWarning: Locale detected as pt_BR - for best results, set to en_US
  warnings.warn(f"Locale detected as {locale} - for best results, set to en_US")

how to solve?

**Comment by neon-ninja** [comment, marked as answer]
Login to your Facebook account with a browser, and change the language of your Facebook account to English. Then pass cookies to the scraper as per the readme.

**Reply by DevSamuelOliveira** [reply]
now i see myself with this problem, ImportError: cannot import name 'get_posts' from partially initialized module 'facebook_scraper' (most likely due to a circular import)

Code:

from facebook_scraper import get_posts

**Reply by neon-ninja** [reply]
The issue is likely that you've named your script `facebook_scraper.py` - name it something else

**Reply by DevSamuelOliveira** [reply]
what an oversight, sorry I'm not familiar with python.

With this module is it possible to scrape pages? or just profile and groups

if it is not possible, will it be soon?

**Reply by neon-ninja** [reply]
Yes, it's possible
